src/main/java/com/atlassian/jira/cloud/jenkins/config/ConfigManagementLink.java

```diff
@@ -122,7 +131,9 @@ public String getRegexFromFormData(final JSONObject formData, final String key)
     }
 
     @VisibleForTesting
-    private void sendConfigDataToJira(final JSONObject formData) throws JiraConnectionFailedException {
+    private void sendConfigDataToJira(final JSONObject formData)
```

Comment:
I don't love the size of this function, its working and tested though, I will create a PR after this one to refactor into smaller methods, just so refactor is not blocking